src/network/util.ts

```diff
@@ -0,0 +1,18 @@
+import {Shard} from "../types";
+import {SHARD_SUBNET_COUNT} from "./constants";
+import {RequestId} from "./codec";
+
+function randomNibble(): string {
+  return Math.floor(Math.random() * 16).toString(16);
+}
+
+export function randomRequestId(): RequestId {
+  return Array.from({length: 16}, () => randomNibble()).join('');
+}
+
+export function shardSubnetAttestationTopic(shard: Shard): string {
+  return `shard${shard % SHARD_SUBNET_COUNT}_attestation`;
```

Comment:
it would be better to move it to constants so we can have all topis in one place. Stub shard number with something like `{shard}` and replace it with real value here